tests/rules/prefix-selectors-with-select.test.ts

```diff
@@ -124,24 +126,6 @@ const invalid: RunTests['invalid'] = [
       ],
     },
   ),
-  fromFixture(
-    stripIndent`
-      export const selectfeature = createSelector((state: AppState) => state.feature)
```

Comment:
We really want to have the prefix `select` followed by a domain.

I haven't seen snake_case been used in the wild, and this is also something I haven't thought of.
For me, it's also OK to use that as a selector name, so `select_feature` would also a valid name.
But I see `selectfeature` still as an invalid name, which should become `selelctFeature` or `select_feature`.
Later, we could also potentially add a config flag for this, e.g. `all | camelCase | snake_case`.

Thoughts?